openedx/core/djangolib/rate_limit_all_user_auth_backend.py

```diff
@@ -0,0 +1,20 @@
+"""
+Overrides the default ratelimitbackend to use the AllowAllUsersModelBackend to preserve pre-1.10 Django functionality
+in regards to authenticating users who are inactive.
+"""
+
+from ratelimitbackend.backends import RateLimitMixin
+
+# TODO: Remove Django 1.11 upgrade shim
+# SHIM: Post-upgrade we should only need the AllowAllUsersModelBackend.
+try:
+    from django.contrib.auth.backends import AllowAllUsersModelBackend as django_backend
```

Comment:
Where is this needed?  We introduced `NEW_USER_AUTH_BACKEND` in `common/djangoapps/student/views.py` specifically to avoid needing to allow inactive users to log in except when they're first created.

Reply:
Ah interesting, I did think we'd addressed it but I didn't see that backend. The errors I was addressing were `common.djangoapps.student.tests.test_login/LoginTest/test_login_not_activated` and `not_activated_no_pii`. They were failing on bad username / pass authenticating. I'll see if I can refactor to use that.